### PR TITLE
add fill_value to RelativeFeatures for when dividing by zero

### LINKED ISSUE #577: RelativeFeatures: add option not to fail when dividing by zero and instead fill inf values with something

### PR DESCRIPTION
**Pull request by solegalli**
closes #653 
closes #577 

FYI @Morgan-Sell 

Making this PR due to rebase conflicts

### CODE REVIEW

File: tests/test_creation/test_relative_features.py [hunk 1]
```diff
@@ -359,19 +359,24 @@ def test_fill_values_when_division_by_zero(
...
+        missing_values="ignore",
     )
 
     X = transformer.fit_transform(df_zero)
 
     new_var = f"Age_{_func[0]}_Marks"
 
     assert X.loc[2, new_var] == _fill_value
+    np.testing.assert_equal(X.loc[1, "Age"], np.nan)
+    np.testing.assert_equal(X.loc[3, "Age"], np.inf)
```
Comment (Morgan-Sell) [inline tests/test_creation/test_relative_features.py:389]: Why did you add the numpy test? I ask out of curiosity.

**Comment by Morgan-Sell** [comment]
hola @solegalli,

I hope one day I'm half the data scientist as Dr. Che!

Sorry for missing the `_floordiv()`, `_truediv()`, and `_mod()` methods. Also, I like how you created the `_find_zeroes_in_reference()` method.

The PR looks great! My only comment/question is due to curiosity.

Lastly, did you discover the root of the rebase conflicts?

File: tests/test_creation/test_relative_features.py (hunk 1 above)
Comment (solegalli) [reply tests/test_creation/test_relative_features.py:389]: To test that only the inf in the intended (newly created) variable are being replaced and not all.

**Comment by solegalli** [comment]
Thank you @Morgan-Sell 

I just rebased on my branch and it worked. My suspicion is that it is related to how you sync (or not) the branches in your fork. You need to sync main and the feature branch in the fork remotely. Then fetch the changes locally. And then rebase. Not sure that's what you do. But that's the only thing I can think of really.
